### DISCUSSION TITLE
Perpetual BEP-1 Tags Discussion

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#11411] [Documentation]
cc @bokeh/dev

This topic is for documenting additions, changes, or removals to `tag:` labels. It is *not* necessary to discuss or debate changes to tags. That may be done at any individual's discretion, just leave a note about the change here when you make it.

**Comment by tcmetzger** [comment]
I'd like to add a `tag: tools` for issues related to the toolbar/tools. There seem to be a lot of issues where this could be applied, like #11372 #11294 #11027 #10654 #11294 or #11275, for example.

**Reply by bryevdv** [reply]
No need to discuss, just add it and document the addition here :)

**Reply by tcmetzger** [reply]
Alright, I added [`tag: tools`](https://github.com/bokeh/bokeh/labels/tag%3A%20tools) to be used for anything related to the toolbar or individual tools!

**Comment by IuryPiva** [comment]
I just added `tag: MathText`

**Comment by tcmetzger** [comment]
I added `tag: accessibility` for issues and PRs related to accessibility improvements.

**Comment by ianthomas23** [comment]
Added `tag: contour`.

**Comment by ianthomas23** [comment]
Added `tag: docker`.

**Comment by mattpap** [comment]
Added `tag: webworker` and `tag: webassembly`. It's implied that they will be used together with `tag: component: bokehjs`.

**Comment by ianthomas23** [comment]
Added `grant: CZI R5` and `grant: SDG 2023` to record work undertaken for these grants.
